assets/app/view/game/route_selector.rb

```diff
@@ -86,13 +86,15 @@ def render
           @selected_route = @routes.first
           store(:routes, @routes, skip: true)
           store(:selected_route, @selected_route, skip: true)
+          @game.round.routes = @routes
         end
 
         if !@selected_route && (first_train = trains[0])
           route = Engine::Route.new(@game, @game.phase, first_train, abilities: @abilities, routes: @routes)
           @routes << route
           store(:routes, @routes, skip: true)
           store(:selected_route, route, skip: true)
+          @game.round.routes = @routes
```

Comment:
why does this need to be done?

Reply:
I need a way to clear the route cache when the user buys route "powers" using SpecialBuy actions through Step::Route. The process_special_buy method needs to clear the cache because the powers bought can/will affect revenue calculations. This change to the UI::route_selector is needed so that process_special_buy can reference the routes being manipulated by the route_selector before the final submit action. I could have added a new method to do this, but round.routes is already accessible.

Reply:
Deleted. UI now clears route cache